app/src/main/java/org/fossasia/pslab/sensorfragment/SensorFragmentMain.java

```diff
@@ -98,34 +102,35 @@ public void onClick(View v) {
                 String itemValue = (String) lvSensor.getItemAtPosition(position);
                 switch (itemValue) {
-                    case "ADS1115":
-                        selectedFragment = SensorFragmentADS1115.newInstance();
-                        break;
-                    case "BH1750":
+                case "ADS1115":
+                    selectedFragment = SensorFragmentADS1115.newInstance();
+                    break;
+                case "BH1750":
+                    if (!howToConnectDialog(itemValue, itemValue, R.drawable.bh1750_schematic))
                         selectedFragment = SensorFragmentBH1750.newInstance();
-                        break;
-                    case "BMP180":
-                        selectedFragment = SensorFragmentBMP180.newInstance();
-                        break;
-                    case "MLX90614":
-                        selectedFragment = SensorFragmentMLX90614.newInstance();
-                        break;
-                    case "HMC5883L":
-                        selectedFragment = SensorFragmentHMC5883L.newInstance();
-                        break;
-                    case "MPU6050":
-                        selectedFragment = SensorFragmentMPU6050.newInstance();
-                        break;
-                    case "SHT21":
-                        selectedFragment = SensorFragmentSHT21.newInstance();
-                        break;
-                    case "TSL2561":
-                        selectedFragment = SensorFragmentTSL2561.newInstance();
-                        break;
-                    case "MPU925x":
-                        selectedFragment = SensorFragmentMPU925X.newInstance();
-                    default:
-                        Toast.makeText(getContext(), "Sensor Not Supported", Toast.LENGTH_SHORT).show();
+                    break;
+                case "BMP180":
+                    selectedFragment = SensorFragmentBMP180.newInstance();
+                    break;
+                case "MLX90614":
+                    selectedFragment = SensorFragmentMLX90614.newInstance();
+                    break;
+                case "HMC5883L":
+                    selectedFragment = SensorFragmentHMC5883L.newInstance();
+                    break;
+                case "MPU6050":
+                    selectedFragment = SensorFragmentMPU6050.newInstance();
+                    break;
+                case "SHT21":
+                    selectedFragment = SensorFragmentSHT21.newInstance();
+                    break;
+                case "TSL2561":
+                    selectedFragment = SensorFragmentTSL2561.newInstance();
+                    break;
+                case "MPU925x":
+                    selectedFragment = SensorFragmentMPU925X.newInstance();
+                default:
+                    Toast.makeText(getContext(), "Sensor Not Supported", Toast.LENGTH_SHORT).show();
```

Comment:
@Vikum94 These are the sensors name right? So can we not put them in the string file with a sensors section.